apps/website/docs/pipelines/azure-login.md

```diff
@@ -5,35 +5,72 @@ sidebar_label: Configuring Azure Login for GitHub Actions
 
 # Configuring Azure Login for GitHub Actions
 
-There are several ways to log into Azure by a GitHub Action. The DX chosen
-method doesn't require maintenance or secret management as it is a passwordless
-approach. Once set up, it just works.
+There are several ways to log into Azure using a GitHub Action. The DX-preferred
+method is passwordless, requiring no maintenance or secret management. Once set
+up, it works seamlessly.
 
-Only two components are needed to let GitHub Actions log into Azure:
+:::tip
+
+The modules `azure_federated_identity_with_github` and
+`azure_github_environment_bootstrap` already implement the steps described in
+this article. While the latter also creates the required GitHub Environments and
+secrets, both modules provide a default set of roles in the current Azure
+subscription, which is likely sufficient for new repositories (more on this
```

Comment:
suggestion: instead of saying "later," please specify which sections of the guide are relevant for DX module users. This will allow us to quickly find the information we need without reading the entire guide.

Reply:
ditto